config/solid_queue.yml

```diff
@@ -1,18 +1,22 @@
-#default: &default
-#   dispatchers:
-#     - polling_interval: 1
-#       batch_size: 500
-#   workers:
-#     - queues: "*"
-#       threads: 5
-#       processes: 1
-#       polling_interval: 0.1
-#
-# development:
-#  <<: *default
-#
-# test:
-#  <<: *default
-#
-# production:
-#  <<: *default
+default: &default
+  dispatchers:
+    - polling_interval: 1
+      batch_size: 500
+      recurring_tasks:
+        data_cleanup:
+          class: DataCleanupJob
+          schedule: every day
```

Comment:
Does this run at a particular time? Do we care what it is?

Reply:
I don't think it's any particular time, but it can run whenever. We just care that old data disappears after some time, so I think this means that we might keep data for up to 91 days, which is fine.